theseus/embodied/__init__.py

```diff
@@ -11,5 +11,7 @@
     DoubleIntegrator,
     GPCostWeight,
     GPMotionModel,
+    HingeLoss,
```

Comment:
Since HingeLoss is derived from CostFunction should we name it HingeCost or just Hinge? The loss part might indicate this is just the error function or similar to robust loss vs robust cost function.

Reply:
Good point about the word loss, but not totally sure about just `Hinge` either. Maybe `HingeCost` is fine for this one?